index.html

```diff
@@ -4832,6 +4833,90 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="mark">
+			<rdef>mark</rdef>
+			<div class="role-description">
+				<p>
+					The mark element represents a run of text in one document marked or highlighted for reference purposes, due to its relevance in another context. When used in a quotation or other block of text referred to from the prose, it indicates a highlight that was not originally present but which has been added to bring the reader's attention to a part of the text that might not have been considered important by the original author when the block was originally written, but which is now under previously unexpected scrutiny. When used in the main prose of a document, it indicates a part of the document that has been highlighted due to its likely relevance to the user's current activity.
```

Comment:
Oops, one more nit/problem: If you look at https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1133.html#role_definitions, you'll notice that there is no short entry. We usually have the first paragraph of any role be a sentence or two max and then elaborate in subsequent paragraphs.

To solve this problem, you could potentially just close the first paragraph after "context."

Sorry for not catching that immediately!!